Yasgui/packages/yasqe/src/index.ts

```diff
@@ -1190,6 +1205,16 @@ export class Yasqe extends CodeMirror {
       this.emit("queryAbort", this, this.req);
       this.updateAbortQueryButton();
     }
+
+    this.abortCountQuery();
+  }
+
+  public abortCountQuery(): void {
+    if (this.countReq) {
+      this.countReq.abort();
+      this.eventService.emitEvent(this.rootEl, "internalCountQueryAbortedEvent", { request: this.countReq });
+      this.emit("countQueryFinished");
```

Comment:
I'm not sure if this is correct here, because isn't it true that aborting the query doesn't actually mean it is really aborted.

Reply:
A signal to abort is triggered on line 1214; subsequently, we notify the clients that this event has occurred.  In case of WB the aborting of the query is not enough an additional BE call is needed to stop ongoing query. This extra call is specific to the WB, which is why an event is fired here purely to notify that a signal for query aborting is present.